mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule MediasoupElixir.MixProject do
   use Mix.Project
 
-  @version "0.10.0"
+  @version "0.11.0-rc.0"
```

Comment:
Why it is RC? Is it RC version?

Reply:
Ah, it has been released at GitHub: https://github.com/versatica/mediasoup/releases
but create is not updated: https://crates.io/crates/mediasoup